RiotSwiftUI/Modules/Common/Theme/ThemeUsersColorsExtension.swift

```diff
@@ -23,7 +23,7 @@ extension ThemeSwiftUI {
     /// Get the stable display name color based on userId.
     /// - Parameter userId: The user id used to hash.
     /// - Returns: The SwiftUI color for the associated userId.
-    func displayNameColor(for userId: String) -> Color {
+    func displayUserColor(for userId: String) -> Color {
```

Comment:
What do you mean by `displayUserColor`? Maybe this kind of signature to be more generic and indicate this is the color associated to this user.
```suggestion
    func userColor(for userId: String) -> Color {
```